frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -1327,11 +1327,11 @@ export function getOngoingUploads(
       {
         host: datastoreHost,
       },
-    )) as Array<OngoingUpload & { dataSourceId: OldDataSourceIdFormat }>;
+    )) as Array<OngoingUpload & { datasetId: OldDatasetIdFormat }>;
     // Rename "team" to "organization" as this is the actual used current naming.
-    return ongoingUploads.map(({ dataSourceId: { name, team }, ...rest }) => ({
+    return ongoingUploads.map(({ datasetId: { name, team }, ...rest }) => ({
```

Comment:
just tested this, and I think this needs to be changed back to dataSourceId here because the back-end uses that. otherwise, I get:

```
TypeError: Cannot read properties of undefined (reading 'name')
```

Reply:
Oh right, thanks for spotting. Should be better now hopefully. will retest on monday

Reply:
tested and worked :)